### PR TITLE
feat(HTB Academy): add presence

### PR DESCRIPTION
**Pull request by Firstoften**
## Description 

Added a new presence for HTB Academy

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Screenshot 2024-01-27 222806
Screenshot 2024-01-27 222843
Screenshot 2024-01-27 222907
Screenshot 2024-01-27 222941

</details>

### CODE REVIEW

**Comment by DarkVillager** [comment]
> * [✔️ ] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
> * [ ✔️] I linted the code by running `yarn format`
> * [ ✔️] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

Change the checkmarks to an 'x' since that makes it checked.

**Comment by Firstoften** [comment]
> > * [✔️ ] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
> > * [ ✔️] I linted the code by running `yarn format`
> > * [ ✔️] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)
> 
> Change the checkmarks to an 'x' since that makes it checked.
> 
> Also if you have any issues with or questions about github feel free to ask in the discord or feel free to dm me.

Did it! Thanks. I'd be very thankful if you answer me :D

File: websites/H/HTB Academy/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+		"id": "339316517558681610",
+		"name": "firstoften"
+	},
+	"service": "HTB Academy",
+	"description": {
+		"en": "HTB Academy is a goldmine of learning opportunities, featuring numerous modules that cover various tools, methodologies, and all aspects of cybersecurity."
+	},
+	"url": "academy.hackthebox.com",
+	"version": "1.0.0",
+	"logo": "https://cdn.rcd.gg/PreMiD/websites/H/HackTheBox/assets/logo.png",
```
Comment (Bas950) [inline websites/H/HTB Academy/metadata.json:13]: please upload it to an external url, so we cant a new cdn url for this presence

File: websites/H/HTB Academy/presence.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+const presence = new Presence({
+		clientId: "1200517025383075840",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	{ pathname } = document.location;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey:
+			"https://cdn.rcd.gg/PreMiD/websites/H/HackTheBox/assets/logo.png",
```
Comment (Bas950) [inline websites/H/HTB Academy/presence.ts:11]: use the new url from metadata url

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/H/HTB Academy/presence.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-one-time-vars/no-one-time-vars */
```
Comment (theusaf) [inline websites/H/HTB Academy/presence.ts:1]: Do not disable eslint rules if possible.

File: websites/H/HTB Academy/presence.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+		const iterminal: HTMLElement = document.querySelector(".iterminal");
+		if (iterminal) {
+			module = document.querySelector(
+				"#layout-wrapper > div.main-content > div > div:nth-child(2) > div > div > h4"
+			);
+		} else {
+			module = document.querySelector(
+				"#layout-wrapper > div.main-content > div > div:nth-child(1) > div > div > h4"
+			);
+		}
```
Comment (theusaf) [inline websites/H/HTB Academy/presence.ts:42]: If possible, see if these selectors can be simplified. For example, `h4` or `.main-content h4`, etc.

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/H/HTB Academy/presence.ts (hunk 4 above)
Comment (Firstoften) [reply websites/H/HTB Academy/presence.ts:42]: This one is not clearly possible. I've tried it while other change request. I'll give it another try though.

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**